test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestMemoryMXBean.java

```diff
@@ -716,33 +719,35 @@ public final void testGetMBeanInfo() {
 		// Now make sure that what we got back is what we expected.
 
 		// Class name
-		AssertJUnit.assertTrue(mbi.getClassName().equals(mb.getClass().getName()));
+		AssertJUnit.assertEquals(mb.getClass().getName(), mbi.getClassName());
 
 		// No public constructors
 		MBeanConstructorInfo[] constructors = mbi.getConstructors();
 		AssertJUnit.assertNotNull(constructors);
-		AssertJUnit.assertTrue(constructors.length == 0);
+		AssertJUnit.assertEquals(0, constructors.length);
 
 		// One public operation (JLM) + five from CILM.
 		MBeanOperationInfo[] operations = mbi.getOperations();
 		AssertJUnit.assertNotNull(operations);
-		AssertJUnit.assertTrue(operations.length == 6);
+		AssertJUnit.assertEquals(6, operations.length);
 
 		// One notification
 		MBeanNotificationInfo[] notifications = mbi.getNotifications();
 		AssertJUnit.assertNotNull(notifications);
-		AssertJUnit.assertTrue(notifications.length == 1);
+		AssertJUnit.assertEquals(1, notifications.length);
 
 		// Print description and the class name (not necessarily identical).
 		logger.debug("MBean description for " + mb.getClass().getName() + ": " + mbi.getDescription());
 
 		// Eight attributes - some writable.
 		MBeanAttributeInfo[] attributes = mbi.getAttributes();
 		AssertJUnit.assertNotNull(attributes);
-		if (javaVersion >= 16) {
-			AssertJUnit.assertTrue(attributes.length == 24);
+		if (javaVersion >= 26) {
+			AssertJUnit.assertEquals(25, attributes.length);
+		} else if (javaVersion >= 16) {
+			AssertJUnit.assertEquals(24, attributes.length);
 		} else {
-			AssertJUnit.assertTrue(attributes.length == 26);
+			AssertJUnit.assertEquals(26, attributes.length);
 		}
 		for (int i = 0; i < attributes.length; i++) {
 			MBeanAttributeInfo info = attributes[i];
```

Comment:
Please replace this with an enhanced for loop:
```java
		for (MBeanAttributeInfo info : attributes) {
```

Reply:
Updated.